core/src/test/java/com/gentics/mesh/core/schema/field/BooleanFieldMigrationTest.java

```diff
@@ -34,20 +32,6 @@ public void testRemove() throws Exception {
 		removeField(CREATEBOOLEAN, FILLTRUE, FETCH);
 	}
 
-	@Test
-	@Override
-	public void testRename() throws Exception {
```

Comment:
Check why this case was removed?

Reply:
A custom script was used for renaming fields.